src/borrow_tracker/tree_borrows/tree/tests.rs

```diff
@@ -598,13 +602,15 @@ mod spurious_read {
         let source = LocStateProtPair {
             xy_rel: RelPosXY::MutuallyForeign,
             x: LocStateProt {
-                state: LocationState::new_init(Permission::new_frozen()),
+                // For the tests, the strongest idempotent foreign access does not matter, so we use `Default::default`
+                state: LocationState::new_init(Permission::new_frozen(), Default::default()),
```

Comment:
```suggestion
                state: LocationState::new_init(Permission::new_frozen(), LastForeignAccess::default()),
```
To make it more clear which type things have. (Same below.)